config/initializers/send_file.rb

```diff
@@ -0,0 +1,2 @@
+public_download_dir = File.join(Rails.public_path, Application.config.x.public_download_folder)
+Dir.mkdir(public_download_dir) unless File.exist?(public_download_dir)
```

Comment:
OK, but to make sure the folder is there in production, can I raise an error( in initialisers ) when folder does not exists ?